### ISSUE TITLE
Autocorrect not working for Lint/UnusedMethodArgument with keyword argument

### ISSUE THREAD

**Issue by cordis-dev** [rubocop-hq/rubocop#6162]
## Expected behavior

Unused argument auto-corrected.

## Actual behavior

Unused argument not auto-corrected.

## Steps to reproduce the problem

Given `.rubocop.yml`:
```yml
AllCops:
  DisabledByDefault: true
  TargetRubyVersion: 2.4

Lint/UnusedMethodArgument:
  Enabled: true
```

and `test.rb`:
```ruby
class Test
  def self.method_with_keyword_arg(test: false)
    return nil;
  end
end
```
run 
```
rubocop --cache false --auto-correct test.rb
```

## RuboCop version

```
$ rubocop -V
0.58.2 (using Parser 2.5.0.3, running on ruby 2.4.4 x64-mingw32)
```

**Comment by Drenmi** [comment]
If we remove the argument, every call site that passes an argument to this method will fail at runtime, so it's not quite safe for us to do so.

**Comment by cordis-dev** [comment]
@Drenmi I was expecting for this argument to _renamed_, not removed. E.g. current behavior for non-keyword arg:
```ruby
class Test
  def self.method_with_arg(test)
    return nil;
  end
end
```

becomes:
```ruby
class Test
  def self.method_with_arg(_test)
    return nil;
  end
end
```

Unless I'm mistaken and you want the remove behavior in future version.

**Comment by Drenmi** [comment]
@cordis-dev: Renaming it would still result in a runtime error, because all the call sites will be referencing the old keyword.
